lib/ilp.js

```diff
@@ -96,10 +102,26 @@ class Ilp {
 
     this.log.info('Connection created successfully')
 
-    await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+    let { success, remainingAmount, error } = await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+
+    // If we have sent NO money and the ilp sender failed, throw an error so we're notified of the failure
+    if (error && remainingAmount === ilpScaledAmount) {
+      throw new Error('ERROR: Encountered error when sending money via ILP: ', error)
```

Comment:
error constructors have weird args (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error); we can either piece-meal construct it (new Error(); err.originalError = error) or just throw the original error and forget the "encountered error when sending money via ILP" -- maybe make that a log instead. lot of options.